src/lsp/cobol_data/data_picture.ml

```diff
@@ -66,6 +66,30 @@ module TYPES = struct
     | Z -> Fmt.char ppf 'Z'
     | Zero -> Fmt.char ppf '0'
 
+  let pp_symbol_cobolized ppf = function
+    | A -> Fmt.char ppf 'A'
+    | B -> Fmt.char ppf 'B'
+    | CR -> Fmt.string ppf "CR"
+    | CS -> Fmt.fmt "$" ppf
+    | DB -> Fmt.string ppf "DB"
+    | DecimalSep -> Fmt.fmt "." ppf
```

Comment:
Sometimes the roles of period and comma is interchanged. But dealing with this would require bringing/keeping the configuration used for interpreting the picture string. Not sure that's worth the hassle.

Same for the currency symbol.